src/api/services/apiKeyService.ts

```diff
@@ -0,0 +1,41 @@
+import { Participant } from '../entities/Participant';
+import { CreatedApiKeyDTO } from './adminServiceHelpers';
+
+export type ApiKeySecretsDTO = {
+  plaintextKey: string;
+  secret: string;
+  name: string;
+};
+
+export const createdApiKeyToApiKeySecrets = (apiKey: CreatedApiKeyDTO): ApiKeySecretsDTO => {
+  // eslint-disable-next-line camelcase
```

Comment:
Is it just me or is this warning not needed?
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/866c498d-d983-4946-8fa6-f7466d0c45d9)

Reply:
Yeah it's not. mb, sorry.